test/rules/no-unused-variable/check-parameters/test.ts.lint

```diff
@@ -65,8 +65,15 @@ export class DestructuringTests {
     }
 }
 
+// Could be a bug with 2.9-dev but getPreEmitDiagnostics isn't
```

Comment:
Can you link to the corresponding TypeScript issue?

Reply:
👍 or if there isn't one, file a ticket with them and link it to this PR!